test/powershell/engine/Module/NewModuleManifest.Tests.ps1

```diff
@@ -58,7 +58,7 @@ Describe "New-ModuleManifest basic tests" -tags "CI" {
         $module.RequiredModules | Should -BeExactly 'PSReadline'
         $module.PrivateData.PSData.ExternalModuleDependencies | Should -BeExactly 'PSReadline'
         $module.PrivateData.PSData.Prerelease | Should -BeExactly 'prerelease'
-        $module.PrivateData.PSData.RequireLicenseAcceptance | Should -BeExactly $true
+        $module.PrivateData.PSData.RequireLicenseAcceptance | Should -BeNullOrEmpty
```

Comment:
Above RequireLicenseAcceptance presents so it should be true here.

Please add TestCases to the test with and without RequireLicenseAcceptance parameter.

Reply:
I reverted that test case and added the null or empty check to the first test where the parameter is not passed.